src/components/checkboxes/checkboxes.yaml

```diff
@@ -220,6 +220,15 @@ examples:
       - value: blue
         text: Blue
 
+- name: with single option set aria-describeby on input
```

Comment:
You can add `readme: false` to your example, since it's only needed for the test.